Gemfile.lock

```diff
@@ -141,7 +141,10 @@ GEM
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
-    ffi (1.17.0)
```

Comment:
I think we still need this one; I found I had to manually add it back in when I tried to docker build. You?

Reply:
I’ve been confused for awhile why sometimes Gemfile lock ends up with multiple versions for different architectures and sometimes it doesn’t. This `ffi` gem looks like it’s still there, but the lock is removing this single one and adding 4 architecture specific ones.

But then the tiktoken_ruby gem below is doing the reverse! This confused me.

Do either of you know which is the “right” way to do it and what causes the flip flopping?

@jlvallelonga was there something you did besides the ruby version change that caused this?

Reply:
oh no I've been reversing this every time I do a bundle install and I missed this one. When I try to start the containers with the different listings in the lock file for each architecture then it fails for me. I started to look it up a while back but didn't get very far. This would be a nice thing to fix for sure. I'll change this part back how it was and hopefully it will work